README.md

```diff
@@ -109,9 +123,18 @@ evm:
   eth-main:
     chain-id: 1
     base-rpc-url: ${ETH_RPC_URL}
-    keyring-pass-env-name: ETH_PASSWORD
+    keyring-pass-env-name: "ETH_PASSWORD"
```

Comment:
why are we changing this? I have it unquoted in my config file

Reply:
reverted.